client/src/requests/api-request.js

```diff
@@ -58,27 +70,27 @@ export const signIn = async (val) => {
 };
 
 export const updateAccountCategory = async (val, token) => {
```

Comment:
Should we take out the token parameters for some of the methods since we're replacing some of the URL's to include {key: "update"} ?

Reply:
done!